### PR TITLE
IA Pages - add traffic data

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
//cc @jdorweiler 
Only available to admins and devs with edit permissions.
screenshot-maria-old duckduckgo com 5001 2015-11-12 16-41-00

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Switched to [Chart.js](http://www.chartjs.org/)
screenshot-maria-old duckduckgo com 5001 2015-11-12 17-29-13

**Comment by jdorweiler** [comment]
The graph looks nice!  I added a script to get the latest traffic data and update the traffic db.

File: script/traffic.pl [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    $traffic > io('traffic.sql');
+
+    system(qq(psql -U ddgc -f traffic.sql) );
+
+};
+
+try{
+    $d->db->txn_do( $update );
+} catch {
+    print "Update error, rolling back\n";
```
Comment (MariagraziaAlastra) [inline script/traffic.pl]: @jdorweiler Does it work for you? I tried running it and I got this error message ^

**Comment by MariagraziaAlastra** [comment]
@jdorweiler I removed script/files.sql. If you were using it somewhere else please create another branch from jd/traffic, before going back to it and pulling the latest changes.

**Comment by MariagraziaAlastra** [comment]
Thanks @jdorweiler! :cactus:

**Pull request merged**